ocw_oer_export/create_csv.py

```diff
@@ -56,6 +56,24 @@ def create_ocw_topic_to_oer_subject_mapping(path=None, file_name=None):
         return {row["OCW Topic"]: row["OER Subject"] for row in reader}
 
 
+def get_cr_sublevel(levels):
+    """Set the value(s) of CR_SUBLEVEL based on the course levels."""
+    level_mappings = {
+        "Undergraduate": ["Community College/Lower Division", "College/Upper Division"],
+        "Graduate": ["Graduate/Professional"],
+        "High School": ["High School", "Community College/Lower Division"],
+        "Non-Credit": ["Career/Technical Education"],
+    }
+    sublevels = [sublevel for level in levels for sublevel in level_mappings.get(level)]
+    return "|".join(sorted(set(sublevels)))
```

Comment:
We recently merged a breaking change to the `run.level` array in MIT Open. It now an array of `{ "code": "id-like-string", "name": "human-readable-string" }` objects. 

This could should be updated to reflect that.

Within MIT Open, the change is not on RC or Prod yet, but it is available locally in `main`. I think it would also be worthwhile to allow changing the API baseurl via an env variable or something so it's easier to develop this against local/RC/prod

Reply:
@ibrahimjaved12 This change is live on RC now.